src/client/table_client.rs

```diff
@@ -1986,6 +2012,20 @@ impl ObTableClientQueryImpl {
             }
         }
 
+        // defense for multiple partition aggreagtion
+        if self.is_aggregation() {
```

Comment:
Make it a query_check() (func name depends on you:D) function.

Reply:
had done.
change the function name to aggregation_check()